src/index.js

```diff
@@ -146,6 +146,7 @@ function disposeStream() {
       subscriber.unsubscribe();
     }
   }
+  process.exit(0);
```

Comment:
hm I don't think there should be a logic like this in a library, at least make it as optional in the disposeStream, I'm sure some users would not want their process to exit unknowingly.